src/Symfony/Component/Security/Core/Authorization/Voter/VoterInterface.php

```diff
@@ -34,6 +36,8 @@ interface VoterInterface
      * @param array $attributes An array of attributes associated with the method being invoked
      *
      * @return int either ACCESS_GRANTED, ACCESS_ABSTAIN, or ACCESS_DENIED
+     *
+     * @deprecated since Symfony 6.2, use {@see getVote()} instead.
```

Comment:
Note sure if is the best way.
Also naming is probably not the best, maybe `voteDecision` is better ?

Reply:
I did not find a better name neither